index.js

```diff
@@ -1 +1,120 @@
-// implement your API here
+const cors = require('cors');
+const express = require('express');
+const server = express();
```

Comment:
an unnecessary but cool little optimization you can do here is to combine your server and express declarations since `express` on line 3 is just a reference to what you captured on line 2, so: `const server = require('express')()` does both things in one line